src/ctap/ctap1.rs

```diff
@@ -309,10 +309,11 @@ impl Ctap1Command {
         let credential_source = decrypt_credential_source(env, key_handle, &application)
             .map_err(|_| Ctap1StatusCode::SW_WRONG_DATA)?;
         if let Some(credential_source) = credential_source {
+            #[allow(unreachable_patterns)]
+            // CTAP1 only supports ECDSA, the default case applies if CTAP2 adds more algorithms.
```

Comment:
I would put the comment before the `#[allow]` otherwise it's a bit confusing. In particular given that this should apply to the match not the let. But expression attribute are not well supported yet (I think).

Reply:
Done